charts/spire/charts/spiffe-oidc-discovery-provider/templates/deployment.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   namespace: {{ include "spiffe-oidc-discovery-provider.namespace" . }}
   labels:
     {{- include "spiffe-oidc-discovery-provider.labels" . | nindent 4 }}
+  {{- if .Values.annotations }}
+  annotations:
+    {{- toYaml .Values.annotations | nindent 4 }}
+  {{- end }}
```

Comment:
The other templates follow a format like this one here from the [spire server statefulset](https://github.com/spiffe/helm-charts/blob/f461a017012a6268d8d1c996841111627b79497c/charts/spire/charts/spire-server/templates/statefulset.yaml#L31-L33). Would you mind changing this up so it matches?

Reply:
```suggestion
  annotations:
    {{- with .Values.annotations }}
      {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
@faisal-memon does line 11 there have too much indention now? seems like it has 6 but nindent is set to 4. might break on multiple annotations?

Reply:
indentation might be off